dll/win32/ws2_32/inc/ws2_32p.h

```diff
@@ -268,6 +268,19 @@ extern DWORD GlobalTlsIndex;
 extern BOOLEAN WsAsyncThreadInitialized;
 extern PWS_SOCK_POST_ROUTINE WsSockPostRoutine;
 
+INT
+WINAPI
+WSACallThread( IN int funcId, IN LPVOID arg1, IN LPVOID arg2);
+
+INT
+WSAAPI
+listen_call(IN SOCKET s,
+       IN INT backlog);
```

Comment:
The 278 line should be slightly indented a bit.

See >> https://imgur.com/a/wpAJY